docs/2-developing/0-connectors/6-using-a-custom-connector.mdx

```diff
@@ -54,40 +88,64 @@ Alternatively, the HTTP API can be used.
 
 When using custom Connectors in a pipeline configuration, they are prefixed with
 phrase `standalone` unlike built in Connectors which are prefixed with
-`builtin`. For example, to use a custom Connector named `file-sync` it would
-look as follows:
-`plugin: standalone:file-sync@[replace-with-connector-version]`.
+`builtin`. Continuing with our [Conduit Snowflake Connector](https://github.com/conduitio-labs/conduit-connector-snowflake) as an example, it would
+look as follows: `plugin: standalone:snowflake@v0.3.1-11-g1dc4b8d`.
 
 More information about referencing connectors can be
 found [here](/docs/using/connectors/referencing).
 
-Below is an example of using a custom Connector in a pipeline configuration file:
+Below is an example of using this same Conduit Snowflake Connector in a [pipeline configuration file](/docs/using/pipelines/configuration-file):
 
 ``` yaml title="pipelines/use-custom-connector.yml"
 version: 2.2
 pipelines:
-  - id: use-custom-connector
+  - id: use-snowflake-connector
     # run pipeline on startup
     status: running
     description: >
-      Example pipeline reading to use a custom file sync connector.
+      Example pipeline reading from Postgres to Snowflake.
 
     connectors:
-      - id: source-sync
+      - id: source-postgres
         type: source
-        # use the custom file-sync plugin as the source
-        plugin: standalone:file-sync@v0.1.0
+        plugin: builtin:postgres
         settings:
-          # use ./data as the directory input to the file-sync connector; the connnector will watch for new files in this directory
-          directory: ./data
+          url: "postgres://exampleuser:examplepass@localhost:5433/exampledb?sslmode=disable"
+          tables: "users"
+          cdcMode: "logrepl"
+          logrepl.publicationName: "examplepub"
+          logrepl.slotName": "exampleslot"
 
-      - id: destination-sync
+      - id: destination-snowflake
         type: destination
-        # use the custom file-sync plugin as the destination
-        plugin: standalone:file-sync@v0.1.0
+        # use the snowflake plugin as the destination
+        plugin: standalone:snowflake@v0.3.1-11-g1dc4b8d
         settings:
-        # use ./new_destination as the directory input to the file-sync connector; the connnector will write files from the pipeline to this directory
-          directory: ./new_destination
+          snowflake.url: "snowflake://exampleuser:examplepass@localhost:5433/exampledb?sslmode=disable"
+          snowflake.username: "exampleuser"
+          snowflake.table: "users"
+          snowflake.orderingColumn: "id"
 ```
 
+:::info
+Here you can find dedicated pages for both of these example connectors to know more about their configuration:
+
+- [Conduit Postgres Connector](/docs/using/connectors/list/postgres)
+- [Conduit Snowflake Connector](/docs/using/connectors/list/postgres)
```

Comment:
This will be updated once the connector.yaml for Snowflake is added.

Reply:
done https://github.com/ConduitIO/conduit-site/pull/248/commits/2e3caf7d6b42a6c686181319541c10a3cdeed3d4